src/lake/Lake/Load/Main.lean

```diff
@@ -131,13 +131,13 @@ def buildUpdatedManifest (ws : Workspace)
   match res with
   | (.ok root, deps) =>
     let ws : Workspace ← {ws with root}.finalize
+    let manifest : Manifest := {name? := ws.root.name, packagesDir? := ws.relPkgsDir}
+    let manifest := deps.foldl (·.addPackage ·.manifestEntry) manifest
+    manifest.saveToFile ws.manifestFile
     LakeT.run ⟨ws⟩ <| ws.packages.forM fun pkg => do
       if let some postUpdate := pkg.postUpdate? then
         logInfo s!"{pkg.name}: running post-update hook"
         postUpdate
```

Comment:
Is the intention here that the post-update hooks might want to rewrite the manifest?

Reply:
Precisely!